README.md

```diff
@@ -20,10 +22,20 @@ optional):
   * tags   - A list of tags describing this package.
   * description - A description of this package.
   * license - The license of the source code in the package.
-  * web    - An optional url for humans to read additional information about
+  * web    - An optional URL for humans to read additional information about
              the package.
+  * doc    - An optional URL for humans to read the package HTML documentation
+
+### Requirements
+
+While we really appreciate your contribution, please follow the requirements: other developers will rely on your package. Non-compliant packages might be removed with no warning.
```

Comment:
I think I know what you're trying to say but it still sounds weird. Just remove the ``: other developers will rely on your package``.